src/components/LinkCardCollection/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Collection } from '@aws-amplify/ui-react';
+
+import LinkCard from '../LinkCard';
+
+const CardData: CardData[] = [
+  {
+    name: 'Card Text1'
+  },
+  {
+    name: 'Card Text2'
+  },
+  {
+    name: 'Card Text3'
+  },
+
+  {
+    name: 'Card Text4'
+  }
+];
+
+export interface CardData {
+  name: string;
+}
+
+export const LinkCardCollection = () => {
+  return (
+    <>
+      <Collection
+        items={CardData}
+        type="list"
+        direction="row"
+        gap="20px"
```

Comment:
```suggestion
        gap="large"
```
Since we'll be using Amplify UI shortly across the entire site, let's get in the habit of using tokens from the design system. https://ui.docs.amplify.aws/react/theming/default-theme/sizes#space

Reply:
Done, thank you